charts/clamav/values.yaml

```diff
@@ -0,0 +1,133 @@
+# Default values for clamav.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# nameOverride -- override the name of the clamav chart
+nameOverride: ""
+
+# fullnameOverride -- override the full name of the clamav chart
+fullnameOverride: ""
+
+service:
+  # service.port -- The port to be used by the clamav service
+  port: 443
+  # service.ingress -- Specifies ingress rules for the clamav service
+  ingress:
+  - podSelector: {}
+
+serviceAccount:
```

Comment:
Since this is only used by `clamav-notify`, it might be better to have it within that section

Reply:
Also used by clamav-notify-cron which has a separate block. I'm also inclined to leave it here since it's probably the number one thing people will want/need to change